detector/aws_elb_duplicate_name.go

```diff
@@ -9,22 +9,20 @@ import (
 
 type AwsELBDuplicateNameDetector struct {
 	*Detector
-	IssueType     string
-	TargetType    string
-	Target        string
-	DeepCheck     bool
 	loadBalancers map[string]bool
 }
 
 func (d *Detector) CreateAwsELBDuplicateNameDetector() *AwsELBDuplicateNameDetector {
-	return &AwsELBDuplicateNameDetector{
+	nd := &AwsELBDuplicateNameDetector{
 		Detector:      d,
-		IssueType:     issue.ERROR,
-		TargetType:    "resource",
-		Target:        "aws_elb",
-		DeepCheck:     true,
 		loadBalancers: map[string]bool{},
 	}
+	nd.Name = "aws_elb_duplicate_name"
+	nd.IssueType = issue.ERROR
+	nd.TargetType = "resource"
+	nd.Target = "aws_elb"
```

Comment:
__[gometalinter]__
* [goconst] 3 other occurrence(s) of "aws_elb" found in: aws_elb_invalid_instance.go:24:14 aws_elb_invalid_security_group.go:24:14 aws_elb_invalid_subnet.go:24:14

<!-- Comment by [SideCI](https://sideci.com) -->